CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Many examples were simplified due to ident normalization changes
 ### Fixed
 - Crash in `kamu login` command on 5XX server responses
+- The push smart protocol now delivers internal errors to the client
```

Comment:
Question for @zaychenko-sergei:

Should we merge and deploy this change (which I assume will include deploying a new version of Jupyter, as we're breaking the protocol), or should we put it on a `smtp-improvements` branch and accumulate further improvements there to be merged and deployed together in a single batch.

Do you think some other improvements that we planned will need breaking protocol changes?

Reply:
We are not winning much from feature branch complications.